Tools/LambdaTestTool-v2/src/Amazon.Lambda.TestTool/Program.cs

```diff
@@ -19,4 +19,17 @@
     config.SetApplicationName(Constants.ToolName);
 });
 
-return await app.RunAsync(args);
+var arguments = new List<string>(args);
```

Comment:
I prefer that you not modify the arguments directly and instead add your logic to https://github.com/aws/aws-lambda-dotnet/blob/feature/lambdatesttool-v2/Tools/LambdaTestTool-v2/src/Amazon.Lambda.TestTool/Commands/RunCommand.cs or https://github.com/aws/aws-lambda-dotnet/blob/feature/lambdatesttool-v2/Tools/LambdaTestTool-v2/src/Amazon.Lambda.TestTool/Commands/Settings/RunCommandSettings.cs

Reply:
Done